### ISSUE TITLE
Support for tabs filling available width

### ISSUE THREAD

**Issue by yokoffing** [black7375/Firefox-UI-Fix#458]
**Is your feature request related to a problem?**

I don't like the default tab width.

**The solution I'd like**

Tabs to fill the available width of the tab bar, similar to how the previous Safari UI worked. 1 tab takes up 100% of the tab bar, 2 tabs take up 50% each, 3 tabs 33%, etc.

**Screenshots or video**

![screenshot-faviconography-mac-2081229744](https://user-images.githubusercontent.com/11689349/187546612-0e1ee77b-9354-4e8e-9b55-2bcb8efbd70c.png)

**Alternatives I've considered**

Similar to https://github.com/MrOtherGuy/firefox-csshacks/blob/master/chrome/tabs_fill_available_width.css, but dropping this into my `userChrome` doesn't work when using Firefox UI Fix. 

**Additional context**

**Comment by black7375** [comment]
Currently applicable as follows:

```css
.tabbrowser-tab[fadein]:not([style^="max-width"]) {
  --tab-max-width: 100vw;
}
```

I will put it as an official option in 7.0.

**Comment by black7375** [comment]
Try to use [`v7.0.0`](https://github.com/black7375/Firefox-UI-Fix/releases/tag/v7.0.0).
